database/artificial_intelligence/NLP.json

```diff
@@ -4,34 +4,34 @@
         "description": "This course teaches  the basics of text processing including basic pre-processing, spelling correction, language modeling, Part-of-Speech tagging, Constituency and Dependency Parsing, Lexical Semantics, distributional Semantics and topic models.",
         "url": "https://archive.nptel.ac.in/courses/106/106/106106211/",
         "category": "ai",
-        "subcategory": "natural-language-processing"
+        "subcategory": "natural_language_processing"
     },
     {
         "name": "Datacamp Tutorials",
         "description": "Provides NLP tutorials for Machine Learning Beginners",
-        "url": "https://www.datacamp.com/blog/what-is-natural-language-processing",
+        "url": "https://www.datacamp.com/blog/what-is-natural_language_processing",
```

Comment:
It's a URL so it should be as it is `https://www.datacamp.com/blog/what-is-natural-language-processing`